linux/jdk/redhat/src/main/packaging/temurin/11/temurin-11-jdk.spec

```diff
@@ -5,7 +5,15 @@
 #  $ rpmdev-vercmp 11.0.13.0.1___7 11.0.13.0.0+8
 #  11.0.13.0.0___8 == 11.0.13.0.0+8
 %global spec_version 11.0.18.0.0.10
+<<<<<<< HEAD
```

Comment:
@chandrashritii Looks like the merges need to be resolved here and in a few of the other files.